### ISSUE TITLE
There is no Korea

### ISSUE THREAD

**Issue by honeymaro** [LucianoGanga/country-codes-list#9]

**Comment by LucianoGanga** [comment]
You're right! 

Thank you for noticing it. Can you help me with the missing data and also correcting if something is wrong? 

```
  {
    "countryNameEn": "North Korea",
    "countryNameLocal": "북조선",
    "countryCode": "KP",
    "currencyCode": "KPW",
    "currencyNameEn": "North Korean won",
    "tinType": "",
    "tinName": "",
    "officialLanguageCode": "ko",
    "officialLanguageNameEn": "Korean",
    "officialLanguageNameLocal": "한국어",
    "countryCallingCode": "850",
    "region": "Asia",
    "flag": "🇰🇵"
  }
```

```
  {
    "countryNameEn": "South Korea",
    "countryNameLocal": "대한민국",
    "countryCode": "KR",
    "currencyCode": "KRW",
    "currencyNameEn": "South Korean won",
    "tinType": "",
    "tinName": "",
    "officialLanguageCode": "ko",
    "officialLanguageNameEn": "Korean",
    "officialLanguageNameLocal": "한국어",
    "countryCallingCode": "82",
    "region": "Asia",
    "flag": "🇰🇷"
  }
```

I'm happy to add it. 

Cheers!
Luciano

**Comment by honeymaro** [comment]
Thanks for your reply.

here's information of Korea.


```json
  {
    "countryNameEn": "North Korea",
    "countryNameLocal": "조선민주주의인민공화국",
    "countryCode": "KP",
    "currencyCode": "KPW",
    "currencyNameEn": "North Korean won",
    "tinType": "",
    "tinName": "",
    "officialLanguageCode": "ko",
    "officialLanguageNameEn": "Korean",
    "officialLanguageNameLocal": "조선어",
    "countryCallingCode": "850",
    "region": "Asia",
    "flag": "🇰🇵"
  }
```

``` json
 {
    "countryNameEn": "South Korea",
    "countryNameLocal": "대한민국",
    "countryCode": "KR",
    "currencyCode": "KRW",
    "currencyNameEn": "South Korean won",
    "tinType": "",
    "tinName": "",
    "officialLanguageCode": "ko",
    "officialLanguageNameEn": "Korean",
    "officialLanguageNameLocal": "한국어",
    "countryCallingCode": "82",
    "region": "Asia",
    "flag": "🇰🇷"
  }
````

**Comment by LucianoGanga** [comment]
Done! It's published at version 1.6.2 :)  

Thanks for your help!
